packages/core/src/browser/saveable.ts

```diff
@@ -209,6 +230,7 @@ export namespace Saveable {
 
     export function apply(
         widget: Widget,
+        isAutoSaveEnabled?: () => boolean,
```

Comment:
If we're trying to centralize the `save` handling, do we still need the whole dance of `Saveable.apply()`, etc.? Why do we retain the option of using a different implementation of `isAutosaveEnabled`?